CRLF line endings in multi line strings

**V version:** 0.1.17 (fdb1433)
**OS:** Linux

**What did you do?**

I tried to see how line endings in multi line strings worked and if different platforms could give different results. I tried to run this code on Linux but with Windows/CRLF line endings

```
a := 'foo
bar'
println(a)
println(a.len)
```

**What did you expect to see?**

To see either 7 or 8 printed depending on if "foo\nbar" or "foo\r\nbar" got generated.
 
**What did you see instead?**

```
$ ./v run test.v
.test.c: In function ‘main’:
.test.c:3502:24: error: missing terminating " character
 3502 |  string a= tos2((byte*)"foo \nbar") ;
      |                        ^~~~
.test.c:3503:1: error: stray ‘\’ ...
(Use `v -debug` to print the entire error message)

V panic: C error. This should never happen. Please create a GitHub issue: https://github.com/vlang/v/issues/new/choose
```

--- a/compiler/parser.v
+++ b/compiler/parser.v
@@ -2814,9 +2814,7 @@ fn (p mut Parser) char_expr() {
 fn format_str(_str string) string {
 	// TODO don't call replace 3 times for every string, do this in scanner.v
 	mut str := _str.replace('"', '\\"')
-	$if windows {
-		str = str.replace('\r\n', '\\n')
-	}
+	str = str.replace('\r\n', '\\n')
 	str = str.replace('\n', '\\n')
 	return str
 }

